var/spack/repos/builtin/packages/kokkos/package.py

```diff
@@ -281,3 +285,47 @@ def cmake_args(self):
         options.append('-DBUILD_SHARED_LIBS=%s' % ('+shared' in self.spec))
 
         return options
+
+    test_script_relative_path = "scripts/spack_test"
+
+    @run_after('install')
+    def setup_build_tests(self):
+        # Skip if unsupported version
+        cmake_source_path = join_path(self.stage.source_path,
```

Comment:
I would do this test via version number. I forget exactly, though, how to test for master/devel.  If someone breaks this by moving directories around, this will silently not run when we would want it to crash.

Reply:
@tldahlgren - Can we check for version during test build?

Reply:
Yes, you should definitely be able to check for the version.  Let me know if you cannot.

Reply:
We generally recommend using `self.spec.satisfies('@[<start>]:[<end>]')` for this purpose, where you can specify the starting version `<start>` and or the ending version `<end>` that applies since it can be used to perform version range checks as well as other constraints on the specification.